src/code-examples/tabs/html.js

```diff
@@ -1,8 +1,8 @@
 const html = `<div class="tabs">
-  <nav role="tablist" aria-labelledby="tablist-1" class="manual">
-    <button id="tab-1" type="button" role="tab" aria-selected="true" aria-controls="tabpanel-1">Section 1</button>
-    <button id="tab-2" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-2" tabindex="-1">Section 2</button>
-    <button id="tab-3" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-3" tabindex="-1">Section 3</button>
+  <nav role="tablist" class="manual">
+    <a id="tab-1" type="button" role="tab" aria-selected="true" aria-controls="tabpanel-1" href="#tabpanel-1">Section 1</a>
+    <a id="tab-2" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-2" tabindex="-1" href="#tabpanel-2">Section 2</a>
+    <a id="tab-3" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-3" tabindex="-1" href="#tabpanel-3">Section 3</a>
```

Comment:
Is there a reason to use links instead of buttons? At the very least, I don't think `type="button"` applies to `a` tags (`type` on an anchor would usually refer to a file MIME type).

I also don't know that focus is (or ever has been) working correctly for the Tabs example. When you use the arrow keys, the displayed tab panel changes, but focus stays on the first tab button, which leads to some weird behavior. We might want an a11y expert to take a look and confirm.

Reply:
It mentions in the "semantic elements" section that for tab elements: "If Tab List is <nav> the Tab element must be <a>, <div>". 

So the `a`'s would be considered tab elements in this case?

Reply:
Yes I didn't know how much we wanted to change this. We can use buttons but we'd have to change the Tab List element to a div and also update our documentation here because there is no mention of the ability to use buttons (but that is what W3C uses in their example https://codepen.io/pen)